### ISSUE TITLE
A11y - Block Post Grid

### ISSUE THREAD

**Issue by BobbyBrooksATX** [ArrayHQ/atomic-blocks#54]
Just an A11y tip/request - 

The Post Grid Block pulls the featured image; if alt text is set on the image correctly, the alt text should be describing the image rather than the purpose of the link.  

The Alt attribute in this case should be set to alt or alt="" so it's null this removes the image from consideration by assistive technologies, then the post title should be added to the link so assistive technologies can describe the link. 

The Read More links need the page title appended to make them unique as well. In the below I added screen reader only span to hide the additional text.  Screen reader will now read - whatever "read more text" set by the block, then append "about + page-title" an assistive technology user will be able to select a unique post from all of the posts on the page and hear in my example "Continue reading about hello world"   

This is also required to pass google lighthouse audit for accessibility 100%. 

`if ( isset( $attributes['displayPostLink'] ) && $attributes['displayPostLink'] ) {
						$list_items_markup .= sprintf(
							'<p><a class="ab-block-post-grid-link ab-text-link" href="%1$s" rel="bookmark">%2$s <span class="sr-only">about %3$s</span></a></p>',
							esc_url( get_permalink( $post_id ) ),
							esc_html( $attributes['readMoreText'] ),
							esc_html( $title )
						);
					}`

**Comment by mikemcalister** [comment]
Fixed in the 1.5.4 release.
